Test1.1/Test1.1/Server.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Test1._1
+{
+    /// <summary>
+    /// класс серва
+    /// </summary>
+    public class Server
+    {
+        private int _port;
+        private readonly CancellationTokenSource _cancellationToken = new CancellationTokenSource();
```

Comment:
```suggestion
        private readonly CancellationTokenSource _cancellationToken = new ();
```
Вы ж не на Java пишете :)